src/ngSanitize/filter/linky.js

```diff
@@ -137,8 +163,19 @@ angular.module('ngSanitize').filter('linky', ['$sanitize', function($sanitize) {
     }
 
     function addLink(url, text) {
+      var key;
+      var computedAttributes = {};
```

Comment:
Do we need to use a new variable `computedAttributes` here?
Can't we just assign `attributes = {}` in an else block starting at [line 177](https://github.com/angular/angular.js/pull/12558/files#diff-c0c7c3cd309480c6baa2b90491d610f3R176)?